packages/rfw/lib/src/dart/binary.dart

```diff
@@ -351,6 +351,8 @@ class _BlobDecoder {
     if (count == 0 && nullIfEmpty) {
       return null;
     }
+    // TODO(dit): Remove ignore below when https://github.com/dart-lang/linter/pull/3279 rolls to stable
+    // ignore:always_specify_types
```

Comment:
Do we know how close this is to rolling through? The reason I hadn't done this is that I assumed it would come through soon enough that it wasn't worth worrying about.

Reply:
I am not 100% sure how that rolls, but the PR above needs to roll to dart, then engine, then flutter and then we need to wait until it's in stable for us to remove the patch? I'd rather ignore+fix later so the rolls can continue, rather than having to merge several 100s of revisions down the line?

(Let me ask the dart team how long does the dart sdk take to pickup a change in the linter)

Reply:
> No, it's a regression on master; it'll be fixed as soon as it hits flutter/flutter master

Ahhhh, then you're right, this should eventually get unbroken by a future roll. Let me then revert the changes in rfw, and just push the changes to go_router that might block that future roll.